examples/webrtc/main.go

```diff
@@ -99,25 +99,27 @@ func main() {
 		}
 	}
 
-	// Set the remote SessionDescription
-	err = peerConnection.SetRemoteDescription(offer)
-	if err != nil {
-		panic(err)
-	}
-
 	// Create an answer
 	answer, err := peerConnection.CreateAnswer(nil)
 	if err != nil {
 		panic(err)
 	}
 
-	// Sets the LocalDescription, and starts our UDP listeners
-	err = peerConnection.SetLocalDescription(answer)
+	
+	// Set the remote SessionDescription
+	err = peerConnection.SetRemoteDescription(offer)
 	if err != nil {
 		panic(err)
 	}
 
-	// Output the answer in base64 so we can paste it in browser
-	fmt.Println(signal.Encode(answer))
+        //Wait for ICE gathering to complete (non-trickle ICE)
```

Comment:
The spaces seem to be off. Could you run `go fmt`?

Reply:
I ran the format on the latest commit